File: projects/poly/code/explode1.js
// return a list of n numbers
// with decreasing values from n to 1
// e.g:
//   explode(10)
// returns:
//   [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
function explode(n) {
  r = [];
  for (let i = 0; i < n; i++) {
    let v = n - i;
    r.push(v);
  }

  return r;
}

console.log(JSON.stringify(explode(11)));

File: projects/punk1/code/explode1.py
# return a list of n numbers with
# increasing value from 1 to n but if n
# is less than 3, then it returns from
# 1 to 6 e.g:
#   explode(10)
# returns:
#   [1,2,3,4,5,6,7,8,9,10]
def explode(n):
    r = []
    if n < 3:
        n = 6
    for i in range(1, n + 1):
        r.append(i)

    return r


print(explode(11))
